util/crc32c.cc

```diff
@@ -371,6 +394,7 @@ uint32_t ExtendImpl(uint32_t crc, const char* buf, size_t size) {
 }
 
 // Detect if SS42 or not.
+#ifndef HAVE_POWER8
```

Comment:
This wrapping ensures that the function isSSE42() is not defined on Power platform.
If we don't add this, then the isSSE42() is defined which does not have relevance.